output_foundry.go

```diff
@@ -200,7 +197,7 @@ func (f *FoundryOutput) Deposit() BaseToken {
 }
 
 func (f *FoundryOutput) StoredMana() Mana {
```

Comment:
Isn't this strange we have "mana" in general interface if not all outputs has mana. I believe removing `StoredMana` from `Output` interface could be problematic in implementation, but at least does not imply all outputs has the ability to store mana.

Reply:
It's a bit strange, but still fine. Delegation and Foundry Outputs don't hold Mana, but since this is only a getter, it can just return 0.